### PR TITLE
Modified scripts for Debian Buster. Added ISO image creation.

### PR DESCRIPTION
**Pull request by Zawoor1**

### CODE REVIEW

File: buster-amd64/bin/dist_iso [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    left = 7%
+    top = 95%-24
+    width = 100%
+    color = "#999" }
+EOF
+
+
+echo -e "\nGenerating iso...\n"
+
+#genisoimage -o ${OUTPUT}/${FILENAME} -b isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot -boot-load-size 4 -boot-info-table "$TEMP_DIR"
```
Comment (ruanda) [inline buster-amd64/bin/dist_iso]: Can you remove commented code?

File: buster-amd64/bin/rootfs_config [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+kernel.panic_on_oops = 1
+EOF
+sed -i 's/" let g:skip_defaults_vim = 1/let g:skip_defaults_vim = 1/g' ${TARGET}/etc/vim/vimrc
+rsync -avS ${TARGET}/etc/skel/ ${TARGET}/root/
+
+
+echo zfs >> ${TARGET}/etc/modules
+
+mkdir ${TARGET}/etc/bash_completion.d
+for i in zfs; do
```
Comment (ruanda) [inline buster-amd64/bin/rootfs_config]: No need for loop here

File: buster-amd64/bin/dist_iso [hunk 3]
```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
```
Comment (ruanda) [inline buster-amd64/bin/dist_iso]: /bin/sh isn't enough?

**Review by ruanda — CHANGES_REQUESTED** [review]

File: buster-amd64/bin/dist_iso (hunk 1 above)
Comment (Zawoor1) [reply buster-amd64/bin/dist_iso]: Done.

File: buster-amd64/bin/rootfs_config (hunk 2 above)
Comment (Zawoor1) [reply buster-amd64/bin/rootfs_config]: It was used before, but it turned out that there was no need to copy additional files. Removed.

File: buster-amd64/bin/dist_iso (hunk 3 above)
Comment (Zawoor1) [reply buster-amd64/bin/dist_iso]: /bin/sh doesn't support echo -e "\n". Changed these to additional echoes.

File: buster-amd64/mkrescue [hunk 4]
```diff
@@ -60,3 +63,5 @@ tar cvf ${OUTPUT}/wmirescue_${ARCH}-netboot.tar -C ${OUTPUT}/netboot .
 ./bin/dist_ipxe ${VERSION} ${OUTPUT} --nofirmware
 ./bin/dist_iso ${VERSION} ${ARCH} ${OUTPUT}
 ./bin/dist_iso ${VERSION} ${ARCH} ${OUTPUT} --nofirmware
+
+umask ${UMASK_ORIG}
```
Comment (ruanda) [inline buster-amd64/mkrescue]: No need to revert umask to orig value as this is the end of the file ;)

Comment (Zawoor1) [reply buster-amd64/mkrescue]: Yeah, you are right. Removed.

**Pull request merged**